Package.swift

```diff
@@ -19,7 +19,8 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMajor(from: "0.3.0")),
         // FIXME: need semver
-        .package(name: "SwiftPM", url: "https://github.com/apple/swift-package-manager.git", .branch("main")),
+        //        .package(name: "SwiftPM", url: "https://github.com/apple/swift-package-manager.git", .branch("release/5.5")),
+        .package(name: "SwiftPM", url: "https://github.com/finestructure/swift-package-manager.git", .branch("accept-private-key-as-data")),
```

Comment:
Oh, I've just noticed that I've opened the PR in SPM against `main` but this repo is tracking `release/5.5`. This will need sorting out if this change is adopted.

Reply:
Changed back. Ignore the comment about which branch to track, I think I got the stray commented out line referencing 5.5 when I was tracking the 5.5 branch of this repo 🤷‍♂️